src/libsystemd/sd-netlink/netlink-message-rtnl.c

```diff
@@ -592,7 +592,7 @@ int sd_rtnl_message_new_link(sd_netlink *rtnl, sd_netlink_message **ret,
                 return r;
 
         if (nlmsg_type == RTM_NEWLINK)
-                (*ret)->hdr->nlmsg_flags |= NLM_F_CREATE | NLM_F_EXCL;
+                (*ret)->hdr->nlmsg_flags |= NLM_F_CREATE | NLM_F_REPLACE;
```

Comment:
Does this work? IIUC the code below, the flag is refused.
https://github.com/torvalds/linux/blob/v6.10/net/core/rtnetlink.c#L3667-L3668